torch/csrc/jit/codegen/cuda/test/test_gpu_utils.cpp

```diff
@@ -3,6 +3,7 @@
 #include <gtest/gtest.h>
```

Comment:
renaming this file to `test_gpu_utils.cpp`. Tests for testing utilities inside `scheduler_utils`, `ir_utils` should go to this file. `test_gpu.cpp` is already too large, and we should avoid adding new tests to it if possible.